lib/curl_log.h

```diff
@@ -124,15 +124,15 @@ void Curl_log_cf_debug(struct Curl_easy *data, struct Curl_cfilter *cf,
  * [CONN-1-0][CF-SSL] this filter very much rocks
  * on connection #1 with sockindex 0 for filter of type "SSL". */
 #define DMSG(d,msg)  \
-  "[CONN-%ld] "msg, (d)->conn->connection_id
+  "[CONN-%zd] "msg, (d)->conn->connection_id
```

Comment:
I think you need to use `"%" CURL_FORMAT_CURL_OFF_T` everywhere for outputting `curl_off_t` variables correctly. The `z` flag is for `size_t` and it can be different than `curl_off_t` !

Reply:
I removed these macros.